modules/life.py

```diff
@@ -0,0 +1,172 @@
+import click
+import chalk
+from config import get_config_file_paths
+import os.path
+from os import listdir
+import time
+import yaml
+from util import *
+
+
+# config file path
+LIFE_CONFIG_FILE_PATH = get_config_file_paths()['LIFE_CONFIG_FILE_PATH']
+LIFE_CONFIG_FOLDER_PATH = get_folder_path_from_file_path(
+    LIFE_CONFIG_FILE_PATH)
+RLIST_PARAMS = ('title', 'author', 'kind', 'tags')
+
+# get file path for today's tasks entry file
+
+def is_in_params(params, query, article):
+    query = query.lower()
+    filter = article[params]
+
+    if type(filter) is list:
+        filter = [ item.lower() for item in filter ]
+    else:
+        filter = filter.lower()
+
+    return (query in filter)
+
+@click.group()
+def life():
+    '''
+        Life command group:\n
+        contains helpful commands to organize your life
+    '''
+
+
+def reading_list_entry_file_path():
+    return LIFE_CONFIG_FOLDER_PATH + '/' + "reading_list.yaml"
```

Comment:
Hey,
Would recommend using `os.path.join` or `os.sep` to make the code platform agnostic.

Reply:
Currently implementing it. Anything else you think that needs to be changed?